irrd/rpsl/tests/test_fields.py

```diff
@@ -231,7 +246,9 @@ def test_rpsl_reference_field():
 
     assert_validation_err(["Invalid AS number", "start with AS-"], field.clean, "RS-1234")
     assert_validation_err(["Invalid AS number", "start with AS-"], field.clean, "RS-1234")
-    assert_validation_err(["Invalid AS number", "at least one component must be an actual set name (i.e. start with AS-"], field.clean, "FOOBAR")
+    assert_validation_err(
+        ["Invalid AS number", "at least one component must be an actual set name (i.e. start with AS-"], field.clean,
```

Comment:
Readability is reduced here. I like the consistency with previous lines in the existing code. And if we do break over multiple lines, at least each argument should get it's own line.